YouTag/ViewController.swift

```diff
@@ -79,7 +79,7 @@ class ViewController: UIViewController, FilterPickerViewDelegate, YYTTagViewDele
 		filterButton.widthAnchor.constraint(equalTo: self.view.widthAnchor, multiplier: 0.2).isActive = true
 		filterButton.heightAnchor.constraint(equalTo: self.view.widthAnchor, multiplier: 0.2).isActive = true
 
-		tagsView = YYTFilterTagView(frame: .zero, tagsList: NSMutableArray(), isDeleteEnabled: true)
+		tagsView = YYTFilterTagView(frame: .zero, tagsList: NSMutableArray(), isAddEnabled: false, isMultiSelection: false, isDeleteEnabled: true, suggestionDataSource: nil)
```

Comment:
Function does not match the init defined for the class + YYTFilterTagView init has to be changed to handle the new `suggestionDataSource` parameter that was introduced to `super.init`